install.sh

```diff
@@ -481,9 +481,9 @@ if [[ "${install_dependencies}" == true ]]; then
   pushd .
     printf "\033[32mBuilding \033[33mgoogletest & lapack\033[32m from source; installing into \033[33m/usr/local\033[0m\n"
     mkdir -p ${build_dir}/deps && cd ${build_dir}/deps
-    ${cmake_executable} -DCMAKE_INSTALL_PREFIX=deps-install -DBUILD_BOOST=OFF ../../deps
-    make -j$(nproc)
-    make install
+    ${cmake_executable} -DCMAKE_INSTALL_PREFIX=deps-install ../../deps
+    make build_deps
```

Comment:
This is so slow compared to my solution in https://github.com/ROCmSoftwarePlatform/rocSOLVER/pull/438.